docs/topics/intellij-idea-plugin-run-configuration.adoc

```diff
@@ -3,13 +3,13 @@
 // * docs/intellij-idea-plugin-guide/master.adoc
 
 [id="intellij-idea-plugin-run-configuration_{context}"]
-= Configuring a run configuration
+= Creating a run configuration
 
 You can configure multiple run configurations to run against each project you import to IntelliJ IDEA.
```

Comment:
The same comment for line 6 applies here and anywhere else you have anything like "configure a run configuration"

Reply:
Fixed.